Dockerfile

```diff
@@ -145,6 +143,94 @@ EOF
 
 WORKDIR /app
 
+####################
+# PHP - FRANKENPHP #
+####################
+
+FROM dunglas/frankenphp:php${PHP_VERSION}-bookworm AS php-franken
+
+LABEL org.opencontainers.image.authors="ambroise@rezo-zero.com"
```

Comment:
Maybe you can add your corporate email @eliot488995568  too. Check `org.opencontainers.image.authors` syntax